tests/cupy_tests/sparse_tests/test_csr.py

```diff
@@ -58,6 +58,28 @@ def test_indptr(self):
         testing.assert_array_equal(
             self.m.indptr, cupy.array([0, 2, 3, 4], self.dtype))
 
+    def test_init_copy(self):
+        n = cupy.sparse.csr_matrix(self.m)
+        cupy.testing.assert_array_equal(n.data, self.m.data)
+        cupy.testing.assert_array_equal(n.indices, self.m.indices)
+        cupy.testing.assert_array_equal(n.indptr, self.m.indptr)
+        self.assertEqual(n.shape, self.m.shape)
+
+    def test_init_copy_other_sparse(self):
+        n = cupy.sparse.csr_matrix(self.m.tocsc())
+        cupy.testing.assert_array_equal(n.data, self.m.data)
+        cupy.testing.assert_array_equal(n.indices, self.m.indices)
+        cupy.testing.assert_array_equal(n.indptr, self.m.indptr)
+        self.assertEqual(n.shape, self.m.shape)
+
+    def test_copy(self):
+        n = self.m.copy()
+        self.assertIsInstance(n, cupy.sparse.csr_matrix)
+        cupy.testing.assert_array_equal(n.data, self.m.data)
```

Comment:
Would you also check that `n.data` and `self.m.data` are copied and not the same instances, as well as `indices` and `indptr`.